compat/types.go

```diff
@@ -0,0 +1,108 @@
+package compat
```

Comment:
This file does not exist in the original finschia-sdk, do you have any references? If so, I would like to know.

Reply:
This file doesn't exist in any of the cosmos-sdk or finschia-sdk and isn't referenced anywhere. The functionalities required for compatibility with Ramus and Finschia-SDK/Ostracon is defined in this source.